rpc/types/events.go

```diff
@@ -0,0 +1,223 @@
+package types
+
+import (
+	"encoding/json"
+	"strconv"
+
+	"github.com/ethereum/go-ethereum/common"
+	ethtypes "github.com/ethereum/go-ethereum/core/types"
+	abci "github.com/tendermint/tendermint/abci/types"
+	evmtypes "github.com/tharsis/ethermint/x/evm/types"
+)
+
+// EventFormat is the format version of the events
+type EventFormat int
+
+const (
+	eventFormatUnknown EventFormat = iota
+
+	// Event Format 1:
```

Comment:
would be good to add a comment when do we use these formats, and why do we use different formats in the first place

Reply:
added some comments to the `EventFormat` type.